src/test/java/com/google/api/codegen/gapic/testdata/csharp/csharp_library.baseline

```diff
@@ -1225,46 +1265,69 @@ namespace Google.Example.Library.V1.Samples
 
 // Generated code. DO NOT EDIT!
 
-// This is a generated sample ("FlattenedAsync", "test_on_success_map")
+// This is a generated sample ("LongRunningFlattenedAsyncPollUntilComplete", "wap")
 
 // sample-metadata
-//   title: test maps in response handling
-//   description: When there is a very long description for a sample file we can split
-//     the description into multiple lines and the samples will be just fine
-//   usage: dotnet run
+//   title: GetBigBook: 'War and Peace'
+//   description: Testing calling forms
+//   usage: dotnet run [--shelf "Novel\\\"\`\b\t\n\r"]
 
-// [START sample]
+using CommandLine;
+
+// [START hopper]
 namespace Google.Example.Library.V1.Samples
 {
     using Google.Example.Library.V1;
+    using Google.Protobuf.WellKnownTypes;
     using System;
-    using System.Threading.Tasks;
+    using System.Collections.Generic;
 
-    public class GetBookAsyncFlattenedAsyncTestOnSuccessMap
+    public class GetBigBookAsyncLongRunningFlattenedAsyncPollUntilCompleteWap
     {
         /// <summary>
-        /// When there is a very long description for a sample file we can split
-        /// the description into multiple lines and the samples will be just fine
+        /// Testing calling forms
         /// </summary>
-        public static async Task SampleGetBookAsync()
+        public static async Task SampleGetBigBookAsync(string shelf)
         {
             LibraryServiceClient libraryServiceClient = await LibraryServiceClient.CreateAsync();
-            BookName name = new BookName("[SHELF_ID]", "[BOOK_ID]");
-            Book response = await libraryServiceClient.GetBookAsync(name);
-            string intKeyVal = response.MapStringValue[123];
-            string booleanKeyVal = response.MapBoolKey[true];
-            int mapValueField = response.MapMessageValue["key"].Field;
-            Console.WriteLine($"Test printing map fields: {response.MapListValueValue["quoted_key"]}");
-            Console.WriteLine($"Test printing enum fields of a map value: {response.MapMessageValue["key"].Alignment}");
+            // string shelf = "Novel\\\"`\b\t\n\r"
+            BookName name = new BookName(shelf, "War and Peace");
+            // Poll until the returned long-running operation is complete
+            Operation<Book, GetBigBookMetadata> operation = await libraryServiceClient.GetBigBookAsync(name);
+            Book response = (await operation.PollUntilCompletedAsync()).Result;
+            // Testing iterating over map fields when both key and value are specified.
```

Comment:
How about  a blank line before line comments?
```
// comment
My.Code();

// comment
My.Code();
```

Reply:
As we've discussed before, it's very hard to know the contents of the previous output handling statement. E.g., If the first output statement is comments, it seems weird to add a blank line above that comment. If there are two consecutive comment statements, it also doesn't seem necessary to insert a blank line between them.